### PR TITLE
Changes to make types confirm to same pattern

### PR DESCRIPTION
**Pull request by silveroliver**

### CODE REVIEW

**Comment by silveroliver** [comment]
Still need to update diagrams.

**Comment by jtweed** [comment]
Location is fine, for license see my comments in #2132.

**Comment by silveroliver** [comment]
Corrected failing files.

**Comment by silveroliver** [comment]
Made License changes based on Jonathans comments.

File: ontologies/Examples/CALM-series.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  "seriesType": {
+    "id": "archive-series",
+    "type": "seriesType",
+    "label": "ArchiveSeries"
+  },
+  "identifiers": [
+    {
+      "value": "9781999809003",
+      "type": "Identifier",
+      "identifierScheme": "ref-no"
```
Comment (alexwlchan) [inline ontologies/Examples/CALM-series.json]: This doesn’t look like the identifier scheme model I was expecting.

File: ontologies/WIP/sierratransformable [hunk 2]
```diff
@@ -27,11 +27,16 @@
    * Example: 
    * "identifiers": [
    * {
-   *   "value": "b11579328",
-   *   "type": "Identifier",
-   *   "identifierScheme": "sierra-system-number"
-   * }
-   * ]
+   *    "indentifierType":
```
Comment (alexwlchan) [inline ontologies/WIP/sierratransformable]: "indentifier"?

File: ontologies/Schema/core-ontology.ttl [hunk 3]
```diff
@@ -59,7 +59,7 @@ wcore:thumbnail rdf:type owl:ObjectProperty ;
     rdfs:range wcore:Location ;
     rdf:isDefinedBy <http://wellcomecollection.org/ontologies/works> .      
 
-wcore:identifierScheme rdf:type owl:ObjectProperty ;
+wcore:identifierType rdf:type owl:ObjectProperty ;
     rdfs:label "identifierScheme"@en ;
```
Comment (alexwlchan) [inline ontologies/Schema/core-ontology.ttl]: This label needs changing.

**Pull request merged**